.travis.yml

```diff
@@ -26,24 +22,16 @@ cache:
     - $HOME/.composer/cache/files
 
 env:
-  matrix: SYMFONY_VERSION=4.0.*
   global:
     - SYMFONY_DEPRECATIONS_HELPER="/.*each.*/"
     - SYMFONY_PHPUNIT_DIR=.phpunit SYMFONY_PHPUNIT_REMOVE="symfony/yaml"
     - KERNEL_CLASS=Symfony\Cmf\Bundle\ResourceRestBundle\Tests\Fixtures\App\Kernel
-    - SYMFONY_PHPUNIT_VERSION=5.7
     - TEST_INSTALLATION=false
 
 matrix:
   include:
-    - php: 7.2
-      env: STABILITY=dev SYMFONY_VERSION=4.0.*
-    - php: 7.1
-      env: STABILITY=dev COMPOSER_FLAGS="--prefer-lowest" SYMFONY_VERSION=2.8.* SYMFONY_DEPRECATIONS_HELPER=weak
-    - php: 7.2
-      env: STABILITY=dev SYMFONY_VERSION=3.3.*
-    - php: 7.2
-      env: STABILITY=dev SYMFONY_VERSION=3.4.*
+    - PHP: 7.2
+      env: COMPOSER_FLAGS="--prefer-lowest"
```

Comment:
the SYMFONY_VERSION constant helps composer to solve the versions because it reduces what it even looks at. maybe if you add that back it works. we probably also need the SYMFONY_DEPRECATIONS_HELPER=weak option to avoid failing because of deprecations (which we don't care about with lowest versions).

Reply:
I would like to resolve the deprecations in a next PR...

Reply:
it is great if you can resolve deprecations that happen with normal builds. for the --prefer-lowest build, we should not try to solve them, it might even be impossible to solve them - i meant to add the deprecations helper constant only in the env of the --prefer-lowest

Reply:
Ok. I got it and will do it.